src/test/console/console-format-applies-skipfiles-to-logged-stacks.txt

```diff
@@ -1,2 +1,2 @@
 logged hello world
- at dont-ignore-me.js:1:1
+ at repl:1:1
```

Comment:
This test probably needs to be changed to not use the repl, because I think the intent is to test that this file is not dropped from the stack.

Reply:
That, or this is an example of the source mapping not working anymore.